### PR TITLE
Add German texts

### PR DESCRIPTION
**Pull request by mlenkeit**

### CODE REVIEW

File: resources/i18n/de/strings.xml [hunk 1]
```diff
@@ -1,4 +1,96 @@
...
+  <string name="ADMISSION_STATE_3G_WITH_PCR_BADGE_TEXT">"3G"</string>
+  <!-- XTXT: Admission State - 3G_WITH_PCR Card Subtitle -->
+  <string name="ADMISSION_STATE_3G_WITH_PCR_SUBTITLE_TEXT">"3G"</string>
+  <!-- XTXT: Admission State - 3G_WITH_PCR Card Body -->
+  <string name="ADMISSION_STATE_3G_WITH_PCR_LONG_TEXT">"Ihre Zertifikate erfüllen die 3G-Plus-Regel. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+
+  <!-- XTXT: Admission State - 3G_WITH_RAT Badge Text -->
+  <string name="ADMISSION_STATE_3G_WITH_RAT_BADGE_TEXT">"3G_WITH_RAT"</string>
+  <!-- XTXT: Admission State - 3G_WITH_RAT Card Subtitle -->
+  <string name="ADMISSION_STATE_3G_WITH_RAT_SUBTITLE_TEXT">"3G_WITH_RAT"</string>
```
Comment (KathrinLuecke) [inline resources/i18n/de/strings.xml:59]: This is not final, right? But "3G mit Schnelltest" would be too long for the badge...However, I fear that "RAT" is not understood by many.

File: resources/i18n/de/strings.xml [hunk 2]
```diff
@@ -1,4 +1,96 @@
...
+  <!-- XTXT: Vaccination State - COMPLETE_IMMUNIZATION Card Subtitle -->
+  <string name="VACCINATION_STATE_COMPLETE_IMMUNIZATION_LONG_TEXT">"Sie haben nun alle derzeit geplanten Impfungen erhalten. Ihr Impfschutz ist vollständig."</string>
+  
+  <!-- XTXT: Vaccination State - COMPLETE_IMMUNIZATION Card Subtitle -->
+  <string name="PERSON_VIEW_2G_CERTIFICATE">"2G-Zertifikat"</string>
+  <!-- XTXT: Vaccination State - COMPLETE_IMMUNIZATION Card Subtitle -->
+  <string name="PERSON_VIEW_TEST_CERTIFICATE">"Testzertifikat"</string>
+
+  <!-- XHED: Booster Notification - Card Title -->
+  <string name="BOOSTER_NOTIFICATION_TITLE_TEXT">"Booster"</string>
```
Comment (KathrinLuecke) [inline resources/i18n/de/strings.xml:92]: I think it should be "Auffrischimpfung" instead of "Booster", as we use "Auffrischimpfung" elsewhere, too.

**Review by KathrinLuecke — CHANGES_REQUESTED** [review]
see comments

File: resources/i18n/de/strings.xml (hunk 1 above)
Comment (mlenkeit) [reply resources/i18n/de/strings.xml:59]: @KathrinLuecke no, this is indeed a typo/copy-paste mistake

File: resources/i18n/de/strings.xml [hunk 3]
```diff
@@ -1,4 +1,96 @@
...
+  <string name="ADMISSION_STATE_2G_PLUS_RAT_LONG_TEXT">"Ihre Zertifikate erfüllen die 2G-Plus-Regel, es sei denn, es wird ein PCR-Test benötigt. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+
+  <!-- XTXT: Admission State - 2G Badge Text -->
+  <string name="ADMISSION_STATE_2G_BADGE_TEXT">"2G"</string>
+  <!-- XTXT: Admission State - 2G Card Subtitle -->
+  <string name="ADMISSION_STATE_2G_SUBTITLE_TEXT">"2G"</string>
+  <!-- XTXT: Admission State - 2G Card Body -->
+  <string name="ADMISSION_STATE_2G_LONG_TEXT">"Ihre Zertifikate erfüllen die 2G-Regel. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+
+  <!-- XTXT: Admission State - 3G_WITH_PCR Badge Text -->
```
Comment (Ein-Tim) [inline resources/i18n/de/strings.xml:49]: Isn't 3G with PCR test known as 3G+?

Comment (mlenkeit) [reply resources/i18n/de/strings.xml:49]: Thanks for catching this, fixed with https://github.com/corona-warn-app/cwa-app-ccl/pull/3/commits/2c66c0a50661c267dc146425506d409550f15180

File: resources/i18n/de/strings.xml (hunk 1 above)
Comment (mlenkeit) [reply resources/i18n/de/strings.xml]: @KathrinLuecke I changed this to how it's currently displayed in 2.16/2.17: 3G for both badge and subtitle text.

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Pull request merged**